component/vshn_postgres.jsonnet

```diff
@@ -38,7 +50,56 @@ local xrd = xrds.XRDFromCRD(
   xrds.LoadCRD('vshn.appcat.vshn.io_vshnpostgresqls.yaml'),
   defaultComposition='vshnpostgres.vshn.appcat.vshn.io',
   connectionSecretKeys=connectionSecretKeys,
-);
+) {
+  spec+: {
+    versions: [
+      v {
+        schema+: {
+          openAPIV3Schema+: {
+            properties+: {
+              spec+: {
+                properties+: {
+                  parameters+: {
+                    properties+: {
+                      size+: {
+                        properties+: {
+                          plan+: {
+                            default: pgParams.defaultPlan,
+                            enum: std.objectFields(pgPlans),
+
+                            description: |||
+                              %s
+
+                              The following plans are available:
+
+                                %s
+                            ||| % [
+                              super.description,
+                              std.join(
+                                '\n\n  ',
+                                [
+                                  '%s - CPU: %s; Memory: %s; Disk: %s' % [ p, pgPlans[p].size.cpu, pgPlans[p].size.memory, pgPlans[p].size.disk ]
+                                  + if std.objectHas(pgPlans[p], 'note') && pgPlans[p].note != '' then ' - %s' % pgPlans[p].note else ''
+
+                                  for p in std.objectFields(pgPlans)
+                                ]
```

Comment:
Not my pretties jsonnet, but it works and will result in proper validation and description in the CRD

Reply:
Better ugly that's working than otherwise :)